<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QPushButton" name="PB1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 92, 100);
font: 48pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(26, 95, 180);
font: 36pt &quot;Ubuntu&quot;;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 92, 100);
font: 48pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 92, 100);
font: 45pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 92, 100);
font: 40pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB9">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB14">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 221, 207);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB15">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(224, 27, 36);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB16">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 97, 203);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB17">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 209, 122);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
